modules/ROOT/pages/bind-api-configure-service-mesh.adoc

```diff
@@ -65,4 +87,88 @@ The binding becomes activate within a few seconds and the API is successfully re
 +
 After you bind your API, the API in API Manager is marked as Active. You are now ready to apply policies to protect your APIs.
 +
-image::api-manager-status-for-service-mesh.png[85%, 85%]
\ No newline at end of file
+image::api-manager-status-for-service-mesh.png[85%, 85%]
+
+[[bind_using_CLI]]
+
+== Binding the Service Using Anypoint Service Mesh CLI
+
+Before you bind the service, ensure that you complete the task prerequisite.
+
+*Task Prerequisite*
+From the API Administration Tab of API Manager, obtain the API name to create the binding using this method.
+
+To create the binding:
+
+. Retrieve the service name from Kubernetes, with which you want to bind the API using the following command:
+`kubectl get svc -n <namespace>`.
+. Verify that your microservice port is named based on xref:https://istio.io/docs/setup/additional-setup/requirements/[Istio requirements].
+. From the Anypoint Service Mesh installer download location, type the following command to bind the adapter with an API:
++
+[source,text,linenums]
+----
+./service-mesh api binding create \
+--name=<binding name> \
+--namespace=<namespace name> \
+--adapter=<adapter name> \
+--api=<api name> \
+--serviceName=<service name>
+----
++
+[%header%autowidth.spread]
+|===
+| Parameter Name | Description | Required/Optional | Default Value
+| name | The unique name that represents the binding of the Kubernetes Service with the API, for example, service-binding-anypoint-security”. 
+ | Required | N/A
+| namespace | The namespace in which your service is deployed. Ensure that you use the same namespace as the adapter you provisioned in the previous procedure. | Required | N/A
+| adapter | Specify the name of the adapter that you provisioned in the previous procedure.  | Required | N/A 
+| api | Specify the name of the API you want to associate with this adapter. You created this API in the previous procedure. | Required | N/A.
+| serviceName | The name of the service that you want to bind to the API, which will be managed by Anypoint Platform. The service name is the identifier of the service that you want to manage with Anypoint Service Mesh. 
+| Required | N/A.
+|===
+
+
+
+[[bind_using_CRD]]
+== Binding the Service Using the CRD File
```

Comment:
You don't necessary use a CRD `File`, you can operate with CRDs without using files.

Reply:
Done! Thanks!